docs/docs-ref-autogen/office/office.settingschangedeventargs.yml

```
@@ -19,7 +19,7 @@ items:
       attempting to save settings at the same time when your add-in is used in a shared (co-authored) document.
 
 
-      **Important:** Your add-in's code can register a handler for the settingsChanged event when the add-in is running
+      **Important**: Your add-in's code can register a handler for the settingsChanged event when the add-in is running
       with any Excel client, but the event will fire only when the add-in is loaded with a spreadsheet that is opened in
       Excel Online, and more than one user is editing the spreadsheet (co-authoring). Therefore, effectively the
```

Again I don't think comma is needed but I could be wrong
```suggestion
      Excel Online and more than one user is editing the spreadsheet (co-authoring). Therefore, effectively the
```